tx/textual/valuerenderer/int_test.go

```diff
@@ -25,32 +25,34 @@ func TestIntJsonTestcases(t *testing.T) {
 	textual := valuerenderer.NewTextual(nil)
 
 	for _, tc := range testcases {
```

Comment:
These tests unfortunately are skewed to already parsed integers and won't catch actual failures like:
* `😊😊😊😊`
* `10838.😊😊`
* `    `

is there a way that we can pass the values directly to the various parsers?

Reply:
> is there a way that we can pass the values directly to the various parsers?

In the Textual working group, we made the decision that Parse only takes valid `Format`-ted strings as inputs. That's because Parse will never be called in actual code, but only in tests to make sure that the `Format`-ted strings are indeed invertible.

My proposal is to keep parse lightweight and assume inputs are well formatted.

Reply:
The other use case I can think of is that if we use it in a signed message, then the entire signed message will be invalid if there are bad formatted strings. So yeah, sounds ok to keep it light. We can always come back and add more validation if needed